src/main/java/io/vertx/pgclient/codec/decoder/message/CloseComplete.java

```diff
@@ -0,0 +1,15 @@
+package io.vertx.pgclient.codec.decoder.message;
+
+import io.vertx.pgclient.codec.Message;
+
+/**
+ * @author <a href="mailto:emad.albloushi@gmail.com">Emad Alblueshi</a>
+ */
+
+public class CloseComplete implements Message {
+
```

Comment:
should make a singleton with private constructor and a `public static final CloseComplete INSTANCE`